src/main/java/com/vaadin/hackathon/git/GitService.java

```diff
@@ -1,40 +1,23 @@
 package com.vaadin.hackathon.git;
 
 import java.io.IOException;
-import java.net.URI;
-import java.net.http.HttpClient;
-import java.net.http.HttpRequest;
-import java.net.http.HttpResponse;
-import java.nio.file.Path;
-import java.time.Duration;
-import java.time.Instant;
-import java.time.OffsetDateTime;
-import java.time.ZoneId;
-import java.time.format.DateTimeFormatter;
 import java.util.ArrayList;
 import java.util.Comparator;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
 import java.util.TreeMap;
 
-import org.eclipse.jgit.api.Git;
-import org.eclipse.jgit.lib.Constants;
-import org.eclipse.jgit.revwalk.RevWalk;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 
-import com.fasterxml.jackson.databind.JsonNode;
-import com.fasterxml.jackson.databind.ObjectMapper;
-
 @Service
 public class GitService {
 
     class VersionComparator implements Comparator<String> {
         @Override
         public int compare(String version1, String version2) {
-            String[] parts1 = version1.substring(1).split("[\\.-]"); // Remove 'v' and split on '.'
-            String[] parts2 = version2.substring(1).split("[\\.-]");
+            String[] parts1 = version1.split("[\\.-]"); //split on '.'
```

Comment:
Seems breaking opening the release notes when clicking on a release

Reply:
`GitHubServer#fetchReleaseNotes` trimmed of the first character of the version assuming it to be a 'v' because the tags were previously malformed.